boards/nucleo-l4r5zi/doc.txt

```diff
@@ -8,6 +8,31 @@
 The Nucleo-L4R5ZI is a board from ST's Nucleo family supporting a ARM Cortex-M4
 STM32L4R5ZI microcontroller with 640KiB of RAM and 2MiB of ROM Flash.
 
+## Hardware
+
+![Nucleo144 L4R5ZI](https://www.st.com/bin/ecommerce/api/image.PF264781.en.feature-description-include-personalized-no-cpn-large.jpg)
+
+### MCU
+
+| MCU          | STM32L4R5ZI                                        |
+|:-------------|:---------------------------------------------------|
+| Family       | ARM Cortex-M4                                      |
```

Comment:
Small nit: the core comes with a FPU
```suggestion
| Family       | ARM Cortex-M4F                                     |
```

Reply:
Thank. I add F to the MCU family and additional row in the table concerning FPU - like in [nucleo-l552ze-q](https://doc.riot-os.org/group__boards__nucleo-l552ze-q.html).